demos/viz_usage.ipynb

```diff
@@ -63,14 +63,14 @@
       "  from pkg_resources import resource_filename\n",
       "0it [00:00, ?it/s]/Users/charleszhang/miniforge3/envs/stac-mjx/lib/python3.11/subprocess.py:1885: RuntimeWarning: os.fork() was called. os.fork() is incompatible with multithreaded code, and JAX is multithreaded, so this will likely lead to a deadlock.\n",
       "  self.pid = _fork_exec(\n",
-      "10it [00:00, 19.80it/s]\n"
```

Comment:
Did you mean to include these changes?

Reply:
I reran it to make sure it worked so the notebook completely updated